streaming/storage/storage.go

```diff
@@ -28,7 +27,7 @@ func NewBadgerStorage(db *badger.DB) *BadgerStorage {
 }
 
 func (bs *BadgerStorage) BeginTransaction() StateTransaction {
-	//bs.db.DropPrefix()
+	// bs.db.DropPrefix()
```

Comment:
If you are modifying the comment you might as well remove this line, I don't really think it's necessary to keep it there

Reply:
Autoformat, but sure